### PR TITLE
Fix sequence additions (#105) 

### PR DESCRIPTION
**Pull request by gabhijit**

### CODE REVIEW

File: examples/tests/18-e2sm.rs [hunk 1]
```diff
@@ -5,5 +5,18 @@ mod e2sm {
 }
 
 fn main() {
+    use asn1_codecs::{aper::AperCodec, PerCodecData};
     eprintln!("E2SM");
+
+    // This will break: Fixed
+    let decode_str = "080000000100000000002043514901200000";
+
+    // This one works if it is uncommented
```
Comment (gth828r) [inline examples/tests/18-e2sm.rs:14]: I'd remove this comment line and the one below it... they are confusing to those without context as long as your changes work.

**Review by gth828r — APPROVED** [review]
I ran the tests and verified that the new one passes for me. I made a comment about some code comments that could be removed, but otherwise I think this is OK. I don't have enough background on the generator, parser, or resolver to provide a full code review in a timely fashion.

File: examples/tests/18-e2sm.rs [hunk 2]
```diff
@@ -5,5 +5,18 @@ mod e2sm {
...
+    //let decode_str = "08000000010000000000204351490120000000";
+
+    let decode_hex = hex::decode(decode_str).unwrap();
+    let mut codec_data = PerCodecData::from_slice_aper(&decode_hex);
+    let e2sm_kpmv3_pdu = e2sm::E2SM_KPM_IndicationMessage::aper_decode(&mut codec_data);
```
Comment (gabhijit) [inline examples/tests/18-e2sm.rs:19]: @gth828r : I will add `encode` function here and add an assertion for the equality of encoded string to the original string, just to make sure codec round trip is fine.

Comment (gabhijit) [reply examples/tests/18-e2sm.rs:19]: This was a good idea! It highlighted a bug in the implementation. I have implemented it locally, once #107 is merged, I will rebase and push these changes.

**Pull request merged**